libnetwork/internal/filelease/filelease_linux.go

```diff
@@ -0,0 +1,19 @@
+package filelease
+
+import (
+	"os"
+	"time"
+
+	"golang.org/x/sys/unix"
+)
+
+func getWriteLease(f *os.File, opts *Opts) (retErr error) {
+	for retry, attempt := true, 0; retry && attempt < opts.attempts; attempt += 1 {
+		if attempt > 0 {
+			time.Sleep(opts.interval)
+		}
+		_, retErr = unix.FcntlInt(f.Fd(), unix.F_SETLEASE, unix.F_WRLCK)
```

Comment:
Ok, yes - that should stop it from logging anything on those platforms. I wasn't sure that was what we'd want, because on Linux we might assume no-log means it got a lease. But, it will prevent any unnecessary concern about what "opened without lease" means, and actual problems that need investigating are very unlikely.